### DISCUSSION TITLE
Konstruktionswerkzeuge - Richtung/Entfernung 3D Angabe in Prozent ermöglichen

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#113] [Ideas]
Bei den Konstruktionswerkzeugen wäre es von Vorteil, wenn bei der Eingabe der Richtung/Entfernung der Winkel auch in Prozent angegeben werden könnte. Diese Angabe wird im Wegebau stark nachgefragt.

![image](https://github.com/gis-eni/webgis-forum/assets/79254257/985e6f6f-fac1-4154-b20e-42a4f8191614)

**Comment by gis-eni** [comment]
Ist grundsätzlich eine gute Idee und wahrscheinlich für die meisten Anwendungsfälle im WebGIS die richtig Option. 
Wird mit Build 5.23.3901 umgesetzt. Außerdem wird das auch gleich der **Default-Wert** sein. Also immer beim öffnen des Dialogs angezeigt werden.

Wer etwas anders möchte kann das ab dem neuen Build über in der ``custom.js`` mit folgendem Flag festlegen:

``` javascript

webgis.usability.measureZAngleDefaultUnit: 'percent',  // deg, gon, percent

```
